s3datarepository/s3datarepository.go

```diff
@@ -66,6 +67,9 @@ func NewDefaultRepository(config map[string]interface{}) (*S3Repository, error)
 		opts = append(opts, WithEndpoint(endpoint))
 	}
 
+	// set default PathPrefix
+	opts = append(opts, WithPathPrefix("/dataset/"))
```

Comment:
maybe it should be `/spinup/dataset/`

Reply:
sure, i'll change it, just wanted something for now